packages/itwin/tree-widget/api/tree-widget-react.api.md

```diff
@@ -421,27 +389,48 @@ export class TreeWidget {
 }
 
 // @public
-export function TreeWidgetComponent(props: SelectableTreeProps): JSX_2.Element;
+export function TreeWidgetComponent(props: TreeWidgetComponentProps): JSX_2.Element | null;
```

Comment:
I reallyl don't like the naming with all those `TreeWidgetSomething`. Maybe we could do something like:
`TreeWidgetComponent` -> `TreeSelector` (probably could make it internal)
`TreeWidgetComponentDefinition` -> `TreeDefinition`
`TreeWidgetComponentProvider` -> `TreeWidgetComponent` (revert to the previous name)